{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,6 +1,52 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "wheel"]
+requires = ["setuptools >= 40.6.0", "wheel", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "{{ cookiecutter.__pypkg }}"
+# Add brief project description
+description = "My package description"
+readme = "README.md"
+keywords = ["one", "two"]
+license = {text = "MIT"}
+license-files = ['LICENSE']
+# Set the minimum version of Python required to run the code
+requires-python = ">=3.9"
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+dynamic = ["version"]
+# Update and version pin all runtime dependencies
+dependencies = [
+    "fastapi",
+    "typer"
```

Comment:
Are these provided as examples or is the idea to save new projects the hassle of adding them? In either case we might want to consider adding a compatibility constraint on the  major version.

Reply:
They're just there to support the example package build. One for a CLI tool (`cli.py`), the other for an API tool (`api.py`).

I didn't originally version pin them because I didn't want to deal with having to maintain the version, but you are correct that having them pinned shows best practice. I'll get that configured.